### PR TITLE
Otel updates for scraping

### PR DESCRIPTION
**Pull request by Kuckkuck**
- bump version
- different label for prometheus scraping
- fixed version for otel-contrib
- 2 years for certs
- enable podMonitor

### CODE REVIEW

File: opentelemetry/chart/values.yaml [hunk 1]
```diff
@@ -44,10 +45,10 @@ openTelemetry:
   cluster:
   region:
   logsCollector:
-    enabled: false
+    enabled: true
   prometheus:
   podMonitor:
-    enabled: false
```
Comment (IvoGoman) [inline opentelemetry/chart/values.yaml:50]: The `logsCollector` and `podMonitor` are also options in the `PluginDefinition` where they are defaulted to true, false respectively.  These values in the `values.yaml` have no effect in this case and the values should be configured on the Plugin instead.

File: opentelemetry/chart/values.yaml [hunk 2]
```diff
@@ -17,14 +17,15 @@ opentelemetry-operator:
     failurePolicy: 'Ignore'
     create: true
     certManager:
-      enabled: false
-    autoGenerateCert:
       enabled: true
+      duration: "17280h"
+    autoGenerateCert:
+      enabled: false
       recreate: false
```
Comment (IvoGoman) [inline opentelemetry/chart/values.yaml:24]: Same as for the comment below. These values are part of the PluginDefinition's `options` and should be configured directly on the Plugin.

File: opentelemetry/chart/values.yaml (hunk 1 above)
Comment (Kuckkuck) [reply opentelemetry/chart/values.yaml]: I have removed these defaults.

File: opentelemetry/chart/values.yaml (hunk 2 above)
Comment (Kuckkuck) [reply opentelemetry/chart/values.yaml:22]: I have removed these defaults.

**Review by IvoGoman — APPROVED** [review]

**Pull request merged**